src/schemes/boundary/open_boundary/boundary_zones.jl

```diff
@@ -90,21 +108,26 @@ bidirectional_flow = BoundaryZone(; plane=plane_points, plane_normal, particle_s
 !!! warning "Experimental Implementation"
     This is an experimental feature and may change in any future releases.
 """
-struct BoundaryZone{BT, IC, S, ZO, ZW, FD, PN}
+struct BoundaryZone{IC, S, ZO, ZW, FD, PN, R}
     initial_condition       :: IC
     spanning_set            :: S
     zone_origin             :: ZO
     zone_width              :: ZW
     flow_direction          :: FD
     plane_normal            :: PN
-    boundary_type           :: BT
+    reference_values        :: R
```

Comment:
```suggestion
    reference_values        :: R
    # Note that the following can't be static type parameters, as all boundary zones in a system
    # must have the same type, so that we can loop over them in a type-stable way.
```